docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -4,22 +4,23 @@ title: Data Grid - Row grouping
 
 # Data Grid - Row grouping [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
 
-<p class="description">Group your rows according to some column values.</p>
+<p class="description">Group rows together based on column values in the Data Grid.</p>
 
-For when you need to group rows based on repeated column values, and/or custom functions.
-In the following example, movies are grouped based on their production `company`:
+The Data Grid Premium provides a row grouping feature which makes it possible to create subsets of rows based on repeated column values or custom functions.
+
+For example, in the demo below, movies are grouped based on their respective values in the production company column:
 
 {{"demo": "RowGroupingBasicExample.js", "bg": "inline", "defaultCodeOpen": false}}
 
 :::info
-If you are looking for row grouping on the server-side, see [server-side row grouping](/x/react-data-grid/server-side-data/row-grouping/).
+This document covers client-side implementation.
+For row grouping on the server side, see [server-side row grouping](/x/react-data-grid/server-side-data/row-grouping/).
```

Comment:
This Vale warning is incorrect 😭 I'll open a separate PR to address it

<img width="1224" alt="Screenshot 2025-02-28 at 9 57 48 AM" src="https://github.com/user-attachments/assets/8dd5a5ee-a5be-4a81-8627-bb4bd78dc35d" />